ntoskrnl/ke/i386/traphdlr.c

```diff
@@ -1357,7 +1313,56 @@ KiTrap0EHandler(IN PKTRAP_FRAME TrapFrame)
             KiEoiHelper(TrapFrame);
```

Comment:
Does this correctly unwind the two function calls KiCheckForSListFault() and KiTrap0EHandler() ?

Reply:
Good catch, will move it outside this function.